ts/Series/Networkgraph/NetworkgraphSeries.ts

```diff
@@ -297,9 +324,41 @@ class NetworkgraphSeries extends Series {
         chart: NetworkgraphChart,
         options: Partial<NetworkgraphSeriesOptions>
     ): NetworkgraphSeries {
-
         super.init(chart, options);
 
+        const dlOptions = this.options.dataLabels;
+
+        // time from init() to the time when dataLabels show up #14398
+        // default set to 2.5 seconds
+        let deferTime = 2500;
+
+        // PROBLEM: if this time is too high
+        // then the dataLabels will never draw because the drawDataLabels()
+        // will not be called after the end of simulation
+        // but how to take care of this edge case?
```

Comment:
In `ReingoldFruhtermanLayout` file you can find `step()` method. If you fire an event there, about a completed simulation, then you can listen to that event in Networgraph. I'm suggesting an event because it wouldn't be the first time we/users need this. We could make this an official API option, listing it among [other events](https://api.highcharts.com/highcharts/series.networkgraph.events).